src/components/bulkConcepts/addBulkConcepts.jsx

```diff
@@ -63,7 +63,7 @@ export class AddBulkConcepts extends Component {
                 value="option2"
               />
               <label className="form-check-label" htmlFor="exampleRadios2">
-                (Other preffered sources here)
+                (Other preferred sources here)
```

Comment:
Is this fix part of this ticket. If not, it should have its own PR. Because if it so happens that this PR is reverted, this change will reappear.

Reply:
@emmabaye, please create it and add the link to it here. Thank you.

Reply:
Here is the ticket, https://issues.openmrs.org/browse/OCLOMRS-272